frontend/src/App.test.tsx

```diff
@@ -4,6 +4,6 @@ import App from "./App"
 
 test("renders Home title", () => {
```

Comment:
oh you've changed what the  test is looking for but not the name of the test, that should be updated to match